.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
+    "react/no-unescaped-entities": "off"
```

Comment:
Why do you need this?

Reply:
Its because eslint by default suggests to use html entities instead of symbols, such as: ', ", >, =, } etc. On my opinion, this rule is not so important so I typically turned it of.
You can read more about this rule [there](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md) and about using entities [there](https://stackoverflow.com/questions/436615/when-should-one-use-html-entities)